client/src/ui/BitbakeCommands.ts

```diff
@@ -188,6 +190,25 @@ async function runTaskCommand (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjec
   }
 }
 
+async function startToasterInBrowser (bitbakeDriver: BitbakeDriver): Promise<void> {
+  const DEFAULT_TOASTER_PORT = 8000
+  const command = 'nohup bash -c "source toaster start"'
```

Comment:
Ideally, "source toaster start" should be a string returned by a compose_* command in BitbakeDriver. That way we keep all commands generation logic in BitbakeDriver if we have to add other global parameters later.

Reply:
done, see if it is ok.